src/components/MediaCard/MediaCard.vue

```diff
@@ -0,0 +1,151 @@
+<template lang="pug">
+LegacyCard
+  div(:class="mediaCardClassName")
+    div(:class="mediaContainerClassName")
+      slot
+    div(:class="infoContainerClassName")
+      Box(:padding="500")
+        BlockStack(:gap="200")
+          InlineStack(
+            align="space-between",
+            :wrap="false",
+            :gap="200",
+          )
+            Text(
+              v-if="typeof title === 'string'",
```

Comment:
It's not correct, it should be `v-if="hasSlot(slots.title)"` => display slot
else, using title prop. 
Check other components for more information